pkgs/by-name/gr/gruvbox-kvantum/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  variant ? "Gruvbox-Dark-Brown",
```

Comment:
I don't know, if other variants will be downloaded from nix cache. How does this work?